app/queries/find_exact_query.rb

```diff
@@ -2,13 +2,15 @@ class FindExactQuery
   class InvalidFindCriteria < StandardError; end
 
   def initialize(
+    content_id:,
```

Comment:
In commit 69becf60384cec34f3e25d8b04ea02afafdfc276, could you add `content_id: nil` to the `find_exact_query_params` in the controller, so that the controller specs pass?  It's a small change, but it would make this commit bisectable..